lib/models/mirror.js

```diff
@@ -3,6 +3,7 @@
 const errors = require('storj-service-error-types');
 const mongoose = require('mongoose');
 const SchemaOptions = require('../options');
+const _ = require('lodash');
```

Comment:
We probably do not need to bring in lodash specifically for `_.isEmpty`, and lodash is not included in package.json, so this will likely fail.